### PR TITLE
runfix: Allow blurred background to be enabled before the call starts

### PR DESCRIPTION
**Pull request by atomrc**
## Description

This PR will allow the video call background blur setting to be saved and reused across calls. 
We, for now, persist the user preference only for the current session. Later the setting will be saved to a persistent storage and allowed to be changed in the audio/video settings. 

This PR also increase the blurring effect.

### CODE REVIEW

File: src/script/media/VideoBackgroundBlur.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ enum FRAMERATE {
 
 const QualitySettings = {
   segmentationModel: SEGMENTATION_MODEL.PERFORMANCE,
-  framerate: FRAMERATE.HIGH,
+  framerate: FRAMERATE.LOW,
```
Comment (atomrc) [inline src/script/media/VideoBackgroundBlur.ts:35]: This should improve performances a bit

File: src/script/media/BackgroundBlurrer/fragmentShader.glsl [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#define BLUR_QUALITY 8 //The higher the value, the more blur. Must be an even number.
+#define BLUR_QUALITY 14 //The higher the value, the more blur. Must be an even number.
```
Comment (atomrc) [inline src/script/media/BackgroundBlurrer/fragmentShader.glsl:1]: This will increase the blurring effect

File: src/script/calling/Participant.ts [hunk 3]
```diff
@@ -18,68 +18,72 @@
...
+    this.hasPausedVideo = pureComputed(() => {
       return this.videoState() === VIDEO_STATE.PAUSED;
     });
-    this.videoStream = ko.observable();
-    this.audioStream = ko.observable();
-    this.isActivelySpeaking = ko.observable(false);
-    this.startedScreenSharingAt = ko.observable();
-    this.isMuted = ko.observable(false);
-    this.isSendingVideo = ko.pureComputed(() => {
+    this.isSendingVideo = pureComputed(() => {
```
Comment (atomrc) [inline src/script/calling/Participant.ts:67]: Those are only changes to make typescript and eslint happier. Nothing interesting to see there

File: src/script/calling/CallingRepository.ts [hunk 4]
```diff
@@ -131,6 +130,7 @@ export class CallingRepository {
   private readonly acceptVersionWarning: (conversationId: QualifiedId) => void;
   private readonly callLog: string[];
   private readonly logger: Logger;
+  private enableBackgroundBlur = false;
```
Comment (atomrc) [inline src/script/calling/CallingRepository.ts:133]: in a further feature, this will be moved to a proper setting in the `A/V` settings

**Review by PatrykBuniX — APPROVED** [review]

**Pull request merged**